docs/regressions-beir-v1.0.0-robust04-multifield.md

```diff
@@ -0,0 +1,69 @@
+# Anserini Regressions: BEIR (v1.0.0) &mdash; robust04
```

Comment:
@justram changed to "Robust04" with caps:
https://github.com/castorini/anserini/blob/master/docs/regressions-beir-v1.0.0-robust04-flat.md

Make consistent?